rxnorm/rxnorm_builder.py

```diff
@@ -48,26 +48,28 @@ def get_umls_data(
         """
         api = umls.UmlsApi(api_key=umls_key)
         metadata = api.get_latest_umls_file_release(
-            target="umls-metathesaurus-full-subset"
+            target="rxnorm-full-monthly-release"
         )
         download_required = False
+
         if not (download_path / metadata["releaseVersion"]).exists():
-            print("New UMLS release available, downloading & updating...")
+            print("New RxNorm release available, downloading & updating...")
             download_required = True
             for version in download_path.iterdir():
                 self.rmtree(version)
             for version in (parquet_path).iterdir():
                 self.rmtree(version)
         if download_required or force_upload:
             api.download_umls_files(
-                target="umls-metathesaurus-full-subset", path=download_path
+                target="rxnorm-full-monthly-release", path=download_path, unzip=False
             )
-        files = list(download_path.glob(f'./{metadata["releaseVersion"]}/META/*.ctl'))
-        filtered_files = []
-        for file in files:
-            if not file.stem.startswith("MRX"):
-                filtered_files.append(file)
-        return filtered_files, download_required, metadata["releaseVersion"]
+            base_utils.unzip_file(
+                download_path / metadata["fileName"], 
+                download_path / metadata["releaseVersion"]
+                )
+            (download_path / metadata["fileName"]).unlink()
+        files = list(download_path.glob(f'./{metadata["releaseVersion"]}/scripts/oracle/*.ctl'))
+        return files, download_required, metadata["releaseVersion"]
```

Comment:
this is the main study specific change - you can see why i was thinking about sharing code between this and UMLS. I still sort of think that's a good idea, but i might wait until I get to cutting over studies to 3.0 to think about doing that.